{"uuid":"ea894190-398d-437e-9669-e23dae87c5cf","historyId":"6727cf18f1ae931fb9b26ec0d9fe1356","fullName":"PersonMapperTest.testInsertPerson","labels":[{"name":"package","value":"PersonMapperTest"},{"name":"testClass","value":"PersonMapperTest"},{"name":"testMethod","value":"testInsertPerson"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"rest-assured-homework"},{"name":"subSuite","value":"PersonMapperTest"},{"name":"host","value":"EJOI-2"},{"name":"thread","value":"6688@EJOI-2.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testInsertPerson","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Failed to initialize database schema","trace":"java.lang.RuntimeException: Failed to initialize database schema\r\n\tat config.DataBaseConfig.initializeSchema(DataBaseConfig.java:58)\r\n\tat config.DataBaseConfig.dbMapper(DataBaseConfig.java:27)\r\n\tat PersonMapperTest.testInsertPerson(PersonMapperTest.java:15)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Attempt to define a second primary key; SQL statement:\nALTER TABLE person ADD PRIMARY KEY (id) [90017-224]\r\n\tat org.h2.message.DbException.getJdbcSQLException(DbException.java:644)\r\n\tat org.h2.message.DbException.getJdbcSQLException(DbException.java:489)\r\n\tat org.h2.message.DbException.get(DbException.java:223)\r\n\tat org.h2.message.DbException.get(DbException.java:199)\r\n\tat org.h2.message.DbException.get(DbException.java:188)\r\n\tat org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:127)\r\n\tat org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:74)\r\n\tat org.h2.command.ddl.AlterTable.update(AlterTable.java:46)\r\n\tat org.h2.command.CommandContainer.update(CommandContainer.java:169)\r\n\tat org.h2.command.Command.executeUpdate(Command.java:256)\r\n\tat org.h2.server.TcpServerThread.process(TcpServerThread.java:413)\r\n\tat org.h2.server.TcpServerThread.run(TcpServerThread.java:191)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n\r\n\tat org.h2.message.DbException.getJdbcSQLException(DbException.java:644)\r\n\tat org.h2.engine.SessionRemote.readSQLException(SessionRemote.java:663)\r\n\tat org.h2.engine.SessionRemote.readException(SessionRemote.java:646)\r\n\tat org.h2.engine.SessionRemote.done(SessionRemote.java:620)\r\n\tat org.h2.command.CommandRemote.executeUpdate(CommandRemote.java:225)\r\n\tat org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:262)\r\n\tat org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)\r\n\tat config.DataBaseConfig.initializeSchema(DataBaseConfig.java:55)\r\n\t... 27 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1738352029388,"stop":1738352030059}